imports/plugins/core/templates/client/templates/email.js

```diff
@@ -99,7 +56,26 @@ Template.emailTemplatesDashboard.events({
 /*
  * Template emailTemplatesSettings Helpers
  */
+Template.emailTemplateSettings.onCreated(function () {
+  this.state = new ReactiveDict();
+
+  this.autorun(() => {
+    const currentData = Template.currentData();
+    const template = Templates.findOne(currentData._id);
```

Comment:
While the use of the variable name "template" is totally logical, I wonder if it doesn't lead to some confusing-to-read code? Just a thought.